data/selinux/snappy.te

```diff
@@ -423,7 +423,7 @@ gen_require(`
 ')
 allow snappy_mount_t proc_t:filesystem { getattr unmount };
 
-allow snappy_mount_t self:capability { sys_chroot sys_admin };
+allow snappy_mount_t self:capability { sys_chroot sys_admin setgid };
```

Comment:
Why are we granting `setgid` here if we're dropping `setgid` as something we need now?

Reply:
Nevermind, I'm dumb. I understand why now. 👍

Reply:
Believe me, it took me forever to finally ask Maciek to explain what was broken :)